File: Second year/Design and Programming Environments/TicketManagerCSharp/Networking/DTOS/TicketDTO.cs
using System;

namespace Networking.DTOS
{
    [Serializable]
    public class TicketDTO
    {
        private int id;
        private int idGame;
        private int reservedSeats;
        private double price;
        private string clientName;

        public TicketDTO(int id, int idGame, int reservedSeats, double price, string clientName)
        {
            this.id = id;
            this.idGame = idGame;
            this.reservedSeats = reservedSeats;
            this.price = price;
            this.clientName = clientName;
        }

        public int Id { get => id; set => id = value; }
        public int IdGame { get => idGame; set => idGame = value; }
        public int ReservedSeats { get => reservedSeats; set => reservedSeats = value; }
        public double Price { get => price; set => price = value; }
        public string ClientName { get => clientName; set => clientName = value; }
    }
}

File: Second year/Design and Programming Environments/TicketManagerJava/Networking/src/main/java/DTOPackage/TicketDTO.java
package DTOPackage;

import java.io.Serializable;

public class TicketDTO implements Serializable {
    private Integer id;
    private Integer idGame;
    private Integer reservedSeats;
    private Double price;
    private String clientName;

    public TicketDTO(Integer id, Integer idGame, Integer reservedSeats, Double price, String clientName) {
        this.id = id;
        this.idGame = idGame;
        this.reservedSeats = reservedSeats;
        this.price = price;
        this.clientName = clientName;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getIdGame() {
        return idGame;
    }

    public void setIdGame(Integer idGame) {
        this.idGame = idGame;
    }

    public Integer getReservedSeats() {
        return reservedSeats;
    }

    public void setReservedSeats(Integer reservedSeats) {
        this.reservedSeats = reservedSeats;
    }

    public Double getPrice() {
        return price;
    }

    public void setPrice(Double price) {
        this.price = price;
    }

    public String getClientName() {
        return clientName;
    }

    public void setClientName(String clientName) {
        this.clientName = clientName;
    }
}
